modules/efk-logging-clo-status.adoc

```diff
@@ -0,0 +1,255 @@
+
+----
+    Elasticsearch Status:
+      Shard Allocation Enabled:  shard allocation unknown
+      Cluster:
+        Active Primary Shards:  0
+        Active Shards:          0
+        Initializing Shards:    0
+        Num Data Nodes:         0
+        Num Nodes:              0
+        Pending Tasks:          0
+        Relocating Shards:      0
+        Status:                 cluster health unknown
+        Unassigned Shards:      0
+      Cluster Name:             elasticsearch
+      Node Conditions:
+        elasticsearch-cdm-mkkdys93-1:
+          Last Transition Time:  2019-06-26T03:37:32Z
+          Message:               0/5 nodes are available: 5 node(s) didn't match node selector.
+          Reason:                Unschedulable
+          Status:                True
+          Type:                  Unschedulable
+        elasticsearch-cdm-mkkdys93-2:
+      Node Count:  2
+      Pods:
+        Client:
+          Failed:
+          Not Ready:
+            elasticsearch-cdm-mkkdys93-1-75dd69dccd-f7f49
+            elasticsearch-cdm-mkkdys93-2-67c64f5f4c-n58vl
+          Ready:
+        Data:
+          Failed:
+          Not Ready:
+            elasticsearch-cdm-mkkdys93-1-75dd69dccd-f7f49
+            elasticsearch-cdm-mkkdys93-2-67c64f5f4c-n58vl
+          Ready:
+        Master:
+          Failed:
+          Not Ready:
+            elasticsearch-cdm-mkkdys93-1-75dd69dccd-f7f49
+            elasticsearch-cdm-mkkdys93-2-67c64f5f4c-n58vl
+          Ready:
+----
+
+This status message indicates that the Elasticsearch CR uses a non-existent PVC.
+
+----
+      Node Conditions:
+        elasticsearch-cdm-mkkdys93-1:
+          Last Transition Time:  2019-06-26T03:37:32Z
+          Message:               pod has unbound immediate PersistentVolumeClaims (repeated 5 times)
+          Reason:                Unschedulable
+          Status:                True
+          Type:                  Unschedulable
+----
+
+This status indicates that the Curator pod cannot be scheduled because the node selector did not match any modes.
+
+----
+Curation:
+    Curator Status:
+      Cluster Condition:
+        curator-1561518900-cjx8d:
+          Last Transition Time:  2019-06-26T03:20:08Z
+          Reason:                Completed
+          Status:                True
+          Type:                  ContainerTerminated
+        curator-1561519200-zqxxj:
+          Last Transition Time:  2019-06-26T03:20:01Z
+          Message:               0/5 nodes are available: 1 Insufficient cpu, 5 node(s) didn't match node selector.
+          Reason:                Unschedulable
+          Status:                True
+          Type:                  Unschedulable
+      Cron Jobs:                 curator
+      Schedules:                 */5 * * * *
+      Suspended:                 false
+----  
+
+This status indicates that the Fluentd pods cannot be scheduled because the node selector did not match any modes.
```

Comment:
A typo: `the node selector did not match any modes` --> `the node selector did not match any nodes`

And I'm not sure if this condition could only caused by the mismatching of nodeSelectors.